css/debug-bar.dev.css

```diff
@@ -327,14 +327,15 @@ body.debug-bar-maximized.debug-bar-visible {
 	vertical-align: top;
 }
 
-#debug-bar-php ol.debug-bar-php-list {
+/* Error notices lists Used by PHP and Deprecated panels. */
+ol.debug-bar-error-notices-list {
```

Comment:
I like getting rid of ID selectors in CSS. We could probably lose the `ol` here too